wolfcrypt/src/ge_448.c

```diff
@@ -10667,25 +10667,24 @@ int ge448_double_scalarmult_vartime(ge448_p2 *r, const uint8_t *a,
                                     const ge448_p2 *p, const uint8_t *b)
 {
 #define GE448_WINDOW_BUF_SIZE 448
+
+#if !defined(WOLFSSL_SMALL_STACK) || (defined(WOLFSSL_NO_MALLOC) && !defined(XMALLOC_USER))
+    int8_t       aslide[GE448_WINDOW_BUF_SIZE];
+    int8_t       bslide[GE448_WINDOW_BUF_SIZE];
+    ge448_p2     pi[16]; /* p,3p,..,31p */
+    ge448_p2     p2;
+    int          i;
+
+    #define PI_PTR(pi) (&(pi))
+#else
     int8_t       *aslide = NULL;
     int8_t       *bslide = NULL;
     ge448_p2     *pi[16]; /* p,3p,..,31p */
     ge448_p2     *p2 = NULL;
-    int          i;
-    int          ret;
-
-#if defined(WOLFSSL_NO_MALLOC) && !defined(XMALLOC_USER)
-    int8_t       aslide_buf[GE448_WINDOW_BUF_SIZE];
-    int8_t       bslide_buf[GE448_WINDOW_BUF_SIZE];
-    ge448_p2     pi_buf[16];
-    ge448_p2     p2_buf;
-
-    aslide = &aslide_buf[0];
-    bslide = &bslide_buf[0];
-    for (i = 0; i < (int)(sizeof pi / sizeof pi[0]); ++i)
-        pi[i] = &pi_buf[i];
-    p2 = &p2_buf;
-#else
+    int          i, ret;
+
+    #define PI_PTR(pi) (pi)
```

Comment:
I think it would be better if pi was declared:
ge448_p2* pi;

An then all 16 were allocated at once.

Reply:
agreed.  will do.